docs/docs/Indexing_JSON.md

```diff
@@ -280,6 +280,87 @@ When JSONPath leads to multiple numerical values:
   - `null` values are skipped
   - Any other value type is causing an indexing failure
 
+## Index JSON arrays as GEO
+
+Starting with RediSearch 2.6.1, search can be done on an array of geo (geographical) values or on a JSONPath leading to multiple geo values.
```

Comment:
Should we use v2.6?

Reply:
Some capabilities are available already with 2.6.0 (multi value TEXT)
Some are available with 2.6.1 (multi value NUMERIC and GEO)